lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -240,6 +240,25 @@ def convert_string(value: str) -> str:
     return value
 
 
+def convert_list(value: list) -> list:
+    """
+    Verify that the parameter value is a list (e.g.,
+    not a JSON dict, or an int), and return it.
+
+    Args:
+        value: REST API request parameter value
+
+    Raises:
+        ConversionError: input can't be converted
+
+    Returns:
+        the input value
+    """
+    if type(value) is not list:
+        raise ConversionError(value, str.__name__, type(value).__name__)
```

Comment:
OK, failed to notice this before: but you're looking for a `list`, not a `str`, so that second parameter should be `list.__name__`.